.gitignore

```diff
@@ -114,3 +114,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
```

Comment:
Should these changes be in a separate pull request?

Reply:
https://github.com/SymbiFlow/fasm/pull/33